script.js

```diff
@@ -17,3 +17,313 @@ for (let i = 0; i < 3; i++) {
     navbar.style.top = "-110vh";
   })
 }
+
+// Modal DOM
+
+const portfolioPopUp = document.createElement('div'); // Create popup div
+portfolioPopUp.className = 'popup';
+portfolioPopUp.innerHTML = `
+<div class="content">
```

Comment:
- [ ] You should have a projects array in your `script.js` file 🗄️. The projects array would be an array of objects containing all your projects and necessary details about the projects. For the avoidance of doubt, each project could be an object in the projects array. 🙏

The projects array should look something like this:
```
const projects = [
  {
    name: "My project",
    description: "My project's description",
    image: "image.png",
    link: "live_link.com",
  }
]
```

- [ ] Nice feature you implemented where the modal disappears if the user clicks outside it 👍 However, the popup modal should be populated with details from your `projects` array. Right now it looks like you are displaying the same modal each time a user clicks the 'see project' button 😎 

- [ ]  I noticed that your projects section is still hardcoded. Please refer to the `projects ` array suggestion above to dynamically display your projects. Loop through the array and use it's contents to display all your projects in the projects section. The advantage of this is that if you have a new project or you want to remove an older project, you can do it in one place only - the projects array. 🙏
https://github.com/V0pe/Portifolio/blob/f07f55d851bf7e17c60f15335bd9b912d60b94b3/index.html#L78

Reply:
Thanks @codepantha. I have adjusted the changes and added the dynamics.